lib/livebook_web/live/hub/edit/personal_component.ex

```diff
@@ -198,7 +209,11 @@ defmodule LivebookWeb.Hub.Edit.PersonalComponent do
 
     on_confirm = fn socket ->
       {:ok, secret} = Livebook.Secrets.update_secret(%Livebook.Secrets.Secret{}, attrs)
-      :ok = Livebook.Hubs.delete_secret(hub, secret)
+
+      case Livebook.Hubs.delete_secret(hub, secret) do
+        :ok -> :ok
+        _ -> raise NotFoundError, secret: secret.name
+      end
```

Comment:
```suggestion
     # In case the secret does not exist, consider it succeeded anyway
     _ = Livebook.Hubs.delete_secret(hub, secret) do
```

Reply:
So now How can we make same exception when delete secret?